core/reporter-api/src/main/java/org/arquillian/reporter/api/builder/report/AbstractReportBuilder.java

```diff
@@ -55,6 +55,20 @@ public BUILDERTYPE addEntry(String entry) {
         return (BUILDERTYPE) this;
     }
 
+
+    @Override
+    public BUILDERTYPE addEntries(List<?> entries) {
+        for (Object entry : entries) {
+            if (entry instanceof Entry) {
+                addEntry((Entry) entry);
+            } else if (entry instanceof String) {
+                addEntry((String) entry);
+            }
```

Comment:
> we are using Java 8 in this project, so please use stream instead of for statements
:+1: 

>if there is another (easy) way of implementing it, please avoid using instanceof as it may have a performance impact (mainly in for cycles)
I think then we need to use reflection to check list type runtime. but I am not sure if it's useful in case of performance.

> the type of the method parameter List<?> is too general. What if you put some object there that is neither an Entry nor a String; then it won't be added into the list of entries at all (without any warning).
I will make it more concrete.

>what I would do is that I would create two methods - one that would take a list of strings and a second one that would take a list of entries.
It's not possible to create two methods due to type erasure of generics.